src/docs/styles/layout/_layout.scss

```diff
@@ -15,98 +15,98 @@ body {
   display: flex;
   flex-direction: column;
   height: 100vh;
-}
 
-main {
-  // TODO: ADS-198 and ADS-297 find a better place for this
-  img {
-    width: 100%;
-    margin-top: space(3x);
-  }
-}
+  .docs-body {
+    display: grid;
+    grid-template-areas: 'nav-secondary' 'main';
+    grid-template-columns: 1fr;
+    grid-template-rows: auto 1fr;
+    flex: 1;
 
-.docs-body {
-  display: grid;
-  grid-template-areas: 'nav-secondary' 'main';
-  grid-template-columns: 1fr;
-  grid-template-rows: auto 1fr;
-  flex: 1;
+    .bsds-nav-secondary {
+      grid-area: nav-secondary;
+    }
 
-  .bsds-nav-secondary {
-    grid-area: nav-secondary;
+    main {
+      display: grid;
+      grid-area: main;
+      grid-template-areas:
+        'intro'
+        'nav-tertiary'
+        'page-content';
+      grid-template-columns: 1fr;
+      grid-template-rows: auto auto 1fr;
+      justify-content: start;
+      justify-items: start;
+      overflow-y: auto;
+      padding: space(4x) calc(space(4x) - space(md));
+
+      // TODO: ADS-198 and ADS-297 find a better place for this
+      img {
+        &:not(.bsds-modal-logo) {
```

Comment:
we probably need a ticket to figure out sizing/styling images in contentful (from markdown). our current solution feels a little fragile.

Reply:
ADS-410